src/main/kotlin/org/jitsi/nlj/transform/node/incoming/RtcpTermination.kt

```diff
@@ -35,55 +35,55 @@ import org.jitsi_modified.impl.neomedia.rtp.TransportCCEngine
 class RtcpTermination(
     private val rtcpEventNotifier: RtcpEventNotifier,
     private val transportCcEngine: TransportCCEngine? = null
-) : Node("RTCP termination") {
+) : TransformerNode("RTCP termination") {
```

Comment:
i think this could also be a filter.  my thought on the distinction is that a filter decides which packets pass through and which don't, but never modify the packets (whereas a transformer does modify).

Reply:
fixed